docs/user-guide/security-features.md

```diff
@@ -1,156 +1,228 @@
 # Security Features
 
+## Introduction
+
 Devtron provides strong security features that help identify vulnerabilities in container images. The system scans container images thoroughly and generates reports if any vulnerabilities are found. 
 
```

Comment:
Please mention as a pre-requisite that a [Vulnerability Scanning Integration (Clair/Trivy)](https://docs.devtron.ai/usage/integrations/clair) must be installed to enable the vulnerability scanning features, as none of the features/options mentioned below will be available if either of the integrations are not installed. 

P.S. Documentation for Trivy integration does not exist as of now. Have added it to the Documentation ToDo sheet.